web/locales/cs.yml

```diff
@@ -0,0 +1,6 @@
+cs:
+  RecurringJobs: Pravidelně opakované
+  Name: Jméno
```

Comment:
But you also added Name. We follow this guide http://guides.rubyonrails.org/i18n.html.